cookbook/request/load_balancer_reverse_proxy.rst

```diff
@@ -95,6 +98,27 @@ That's it! It's critical that you prevent traffic from all non-trusted sources.
 If you allow outside traffic, they could "spoof" their true IP address and
 other information.
 
+My Reverse Proxy sends X-Forwarded-For but does not filter the Forwarded header
+-------------------------------------------------------------------------------
+
+Many popular proxy implementations do not yet support the Forwarded header and
+does not filter it by default configuration. Ideally, you would configure this
```

Comment:
"proxy implementations" is plural, so this needs to be "do not filter it by default." (I would skip configuration here as well)